content/en/dashboards/guide/slo_data_type.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Graph historical SLO data on Dashboards
+kind: guide
+disable_toc: false
+---
+
```

Comment:
Could we also add a banner up here that says "This feature is currently in public beta."?